targets/TARGET_NORDIC/TARGET_NRF5x/qspi_api.c

```diff
@@ -193,26 +197,25 @@ qspi_status_t qspi_prepare_command(qspi_t *obj, const qspi_command_t *command, b
 qspi_status_t qspi_init(qspi_t *obj, PinName io0, PinName io1, PinName io2, PinName io3, PinName sclk, PinName ssel, uint32_t hz, uint8_t mode)
 {
     (void)(obj);
-    if (hz > MBED_HAL_QSPI_MAX_FREQ) {
-        return QSPI_STATUS_INVALID_PARAMETER;
-    }
+		if (hz > MBED_HAL_QSPI_MAX_FREQ)
+			return QSPI_STATUS_INVALID_PARAMETER;
 
     // memset(config, 0, sizeof(config));
-
-    config.pins.sck_pin = (uint32_t)sclk;
+	
+	  config.pins.sck_pin = (uint32_t)sclk;
```

Comment:
Please do not introduce new spaces (if possible set your editor to do this for you on lines you edit)